packages/flutter/lib/src/widgets/dual_transition_builder.dart

```diff
@@ -0,0 +1,200 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
```

Comment:
We shouldn't import the widgets package in this file since this file is also in that same package. Instead, we should import every file that's needed independently [like this](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/heroes.dart#L7).